src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -331,6 +332,22 @@ export const ArtworkFilterContextProvider: React.FC<
     },
   }
 
+  // If pathname changes, reset the filter state
+  const { router, match } = useRouter()
+  const currentPathname = match?.location?.pathname
+
+  useEffect(() => {
+    if (!router || !currentPathname) return
+
+    router.addNavigationListener(({ pathname: nextPathname }) => {
+      if (currentPathname !== nextPathname) {
+        artworkFilterContext.resetFilters()
+      }
```

Comment:
wow, nice, not so bad 👍

Reply:
oh awesome! i didn't think of throwing a `useEffect` right in here.